csharp/src/SeedLang/Engine.cs

```diff
@@ -18,21 +18,27 @@
 using SeedLang.Ast;
 using SeedLang.Common;
 using SeedLang.Interpreter;
-using SeedLang.Runtime;
 using SeedLang.X;
 
 namespace SeedLang {
   // Engine is the facade class of the SeedLang core library. It provides interfaces of compiling
   // and executing SeedX programs, registering visualizers, and inspecting the execution status of
   // programs.
   public class Engine {
+    // The flag to indicate if variable tracking is enabled. It's enabled by calling
+    // "EnableVariableTracking", or registering variable related visualizers like "VariableDefined",
+    // "VariableDeleted" etc.
+    public bool VariableTrackingEnabled => _vm.VisualizerCenter.VariableTrackingEnabled;
```

Comment:
```suggestion
    public bool IsVariableTrackingEnabled => _vm.VisualizerCenter.VariableTrackingEnabled;
```

to be consistent with the following names such as "IsProgramCompiled"?

Reply:
Done.